manifests/alias/range.pp

```diff
@@ -38,28 +38,16 @@
 # Copyright (C) 2013 Mike Arnold, unless otherwise noted.
 #
 define network::alias::range (
-  $ensure,
-  $ipaddress_start,
-  $ipaddress_end,
-  $clonenum_start,
-  $noaliasrouting = false,
-  $restart = true,
-  $netmask = false,
-  $broadcast = false,
-  $arpcheck = true,
+  Enum['up', 'down', 'absent'] $ensure,
+  IP::Address::V4::NoSubnet $ipaddress_start,
+  IP::Address::V4::NoSubnet $ipaddress_end,
+  String $clonenum_start,
+  Boolean $noaliasrouting = false,
+  Boolean $restart = true,
+  Optional[IP::Address::V4::NoSubnet] $netmask = undef,
+  Optional[IP::Address::V4::NoSubnet] $broadcast = undef,
```

Comment:
Previous default values were _wrong_. This change has no impact on the way templates use the data.